src/ProtectedRoomsSet.ts

```diff
@@ -210,15 +215,9 @@ export class ProtectedRoomsSet {
     }
 
     /**
-     * Sync all the rooms with all the watched lists, banning and applying any changed ACLS.
-     * @param verbose Whether to report any errors to the management room.
+     * Synchronize all the protected rooms with all of the policies described in the watched policy lists.
      */
-    public async syncLists(verbose = true) {
-        for (const list of this.policyLists) {
-            const changes = await list.updateList();
-            await this.printBanlistChanges(changes, list);
-        }
-
+    public async syncRoomsWithPolicies() {
```

Comment:
At a glance, the difference between `syncLists` and `syncRoomWithPolicies` isn't clear. That could work if the latter was a `private` method, but since it's public, I believe that we need a more explicit name.